_i18n/en.yml

```diff
@@ -24,7 +24,7 @@ home:
     description: We organize for activism, civic engagement and education in the Bay Area, Seattle, across the United States and around the world.
   who_we_support:
     title: Who we support
-    description: We work in solidarity with existing movements towards social justice, workers' rights, and economic inclusion.
+    description: We work in solidarity with existing movements towards social justice, workers' rights, and economic inclusion. Black Lives Matter!
```

Comment:
I'd remove Black Lives Matter here. It's weirdly out of context, and looks tokenizing/cheap. Thoughts?

Reply:
Yeah, I was worried about that myself. I just felt bad about removing it entirely.